src/main/java/in/twizmwaz/cardinal/module/modules/ctf/post/PostBuilder.java

```diff
@@ -0,0 +1,89 @@
+import in.twizmwaz.cardinal.util.Numbers;
+import in.twizmwaz.cardinal.util.Strings;
+import in.twizmwaz.cardinal.util.Teams;
+import org.bukkit.Bukkit;
+import org.bukkit.craftbukkit.libs.jline.internal.Log;
+import org.jdom2.Element;
+
+import java.util.List;
+
+public class PostBuilder implements ModuleBuilder {
+
+    @Override
+    public ModuleCollection<? extends Module> load(Match match) {
+        ModuleCollection<Post> results = new ModuleCollection<>();
+        for (Element flags : match.getDocument().getRootElement().getChildren("flags")) {
+            for (Element post : flags.getChildren("post")) {
+                Post postp = parsePostElement(post);
+                Log.info("Added postp " + postp.getId());
+                results.add(postp);
+            }
+            for (Element subFlags : flags.getChildren("flags")) {
+                for (Element post : subFlags.getChildren("post")) {
+                    Post postp = parsePostElement(post);
+                    Log.info("Added postp " + postp.getId());
+                    results.add(postp);
+                }
+            }
+        }
+        return results;
+    }
+
+    public static Post getPost(String id) {
+        for (Post post : GameHandler.getGameHandler().getMatch().getModules().getModules(Post.class)) {
+            if (post.getId().toLowerCase().equals(id.toLowerCase())) return post;
+        }
+        return null;
+    }
+
+    public static Post parsePostElement(Element element) {
+        Post post = null;
+        if (element.getName().toLowerCase().equals("post")) {
+            List<RegionModule> regions = Lists.newArrayList();
+            if (element.getText() != null) {
+                regions.add(new PointRegion(new PointParser(element)));
+            } else {
+                for (Element e : element.getChildren()) {
+                    regions.add(RegionModuleBuilder.getRegion(e));
+                }
+            }
+            String id = element.getAttributeValue("id") == null ? null : element.getAttributeValue("id");
+            TeamModule owner = element.getAttributeValue("owner") == null ? null : Teams.getTeamById(element.getAttributeValue("owner")).orNull();
+            boolean permanent = Boolean.parseBoolean(element.getAttributeValue("permanent", "false"));
+            boolean sequential = Boolean.parseBoolean(element.getAttributeValue("sequential", "false"));
+            int pointsRate = Numbers.parseInt(element.getAttributeValue("points-rate", "0"));
+            FilterModule pickupFilter;
+            if (element.getAttributeValue("pickup-filter") != null) {
+                pickupFilter = FilterModuleBuilder.getFilter(element.getAttributeValue("pickup-filter"));
+            } else {
+                pickupFilter = FilterModuleBuilder.getFilter(element.getChild("pickup-filter"));
+            }
+            int recoverTime = Strings.timeStringToSeconds(element.getAttributeValue("return-time", "30s"));
```

Comment:
Yes it's `recover-time` but I've seen `return-time` in all XML I checked. I don't what exactly what I need to do because all CTF XML will be broken if I let `recover-time`.